.github/stale.yml

```diff
@@ -1,25 +1,33 @@
 # Number of days of inactivity before an issue becomes stale
 daysUntilStale: 30
+
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 14
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels:
+  - "more info requested"
+
 # Issues with these labels will never be considered stale
-exemptLabels:
-  - backlog
-  - epic
-  - pinned
-  - security
-  - triage
+exemptLabels:[]
+#  - backlog
+#  - epic
+#  - pinned
+#  - security
+#  - triage
```

Comment:
Ideally, these should be removed from the file unless there's a reason we expect to need to re-enable them in the near term.